External/Notepad/Theme.swift

```diff
@@ -17,7 +17,7 @@ class Theme: NSObject {
 
     /// Default Font Size
     ///
-    private static let defaultFontSize = CGFloat(14)
+    private static let defaultFontSize = CGFloat(SimplenoteConstants.NoteFontSizeNormal)
```

Comment:
It'd probably be a good idea to drop this constant, since it's a constant of a constant.

Plus: noted a bit below, since there are a bunch of constants named `NoteFontSize`, we could probably group them into their own namespace. Perhaps `FontSettings.defaultFontSize` ? 

Or... `FontConstants`, but `FontSettings` somehow feels cooler?